src/app_service/service/token/service.nim

```diff
@@ -282,7 +282,9 @@ QtObject:
       if tokensResult.isNil or tokensResult.kind == JNull:
         raise newException(Exception, "Error in response of getting supported tokens list")
 
-      let tokenList =  Json.decode($tokensResult, TokenListDto, allowUnknownFields = true)
+      let tokenResultStr = $tokensResult
```

Comment:
please add a comment explaining what this is supposed to do so nobody removes it by mistake